src/OrchardCore/OrchardCore.ContentManagement.Display/ContentItemDisplayManager.cs

```diff
@@ -57,6 +57,10 @@ public async Task<IShape> BuildDisplayAsync(ContentItem contentItem, IUpdateMode
             }
 
             var contentTypeDefinition = _contentDefinitionManager.GetTypeDefinition(contentItem.ContentType);
+            if (contentTypeDefinition == null)
+            {
+                throw new ArgumentNullException(contentItem.ContentType);
```

Comment:
Right, so here just log an error and don't render anything. Or throw an OrchardException with a meaningful error message.

Reply:
Logger is not referenced, is it ok to add it?

Reply:
No problem if you reference logging abstractions